dcos/http.py

```diff
@@ -86,7 +114,11 @@ def _request(method,
     if 'headers' not in kwargs:
         kwargs['headers'] = {'Accept': 'application/json'}
 
-    verify = _verify_ssl(verify, toml_config)
+    if _is_request_to_dcos(url, toml_config):
+        verify = _verify_ssl(verify, toml_config)
+    else:
+        # Ignore core.ssl_verify if request is outside the DC/OS cluster (DCOS_OSS-618)
+        verify = None
```

Comment:
Here `verify` can be a path, and even though I coulnd't see it in the codebase, my concern was if this function gets called with an explicit custom CA path for the cluster and an external https url. In such cases I wanted to force verify to `None`.